server/routes/contacts/manage/start/startManageContactsJourneyController.test.ts

```diff
@@ -0,0 +1,139 @@
+import type { Express } from 'express'
+import request from 'supertest'
+import { SessionData } from 'express-session'
+import { v4 as uuidv4 } from 'uuid'
+import { appWithAllRoutes, user } from '../../../testutils/appSetup'
+import AuditService, { Page } from '../../../../services/auditService'
+import ManageContactsJourney = journeys.ManageContactsJourney
```

Comment:
for consistency could we use `import { ManageContactsJourney } from '../../../../journeys'; `